### PR TITLE
cli/verbs.go: Write a Job Report

### PR DESCRIPTION
**Pull request by glabelderp**
This will create a json file under /logs/job.result, which firmwareci
  can pull out of the docker container and parse.

Signed-off-by: Jo <johannes.bartunek@9elements.com>

### CODE REVIEW

File: cli/verbs.go [hunk 1]
```diff
@@ -85,6 +85,23 @@ func run(requestor string, transport transport.Transport, stdout io.Writer) erro
...
 			}
+
+			//write the result into it's own file so that firmwareci
+			//can more easily parse the results
+			buffer.Reset()
+			if err := encoder.Encode(resp); err != nil {
+				return err
+			}
+
+			jobLog, err := os.Create("/logs/job.result")
```
Comment (walterchris) [inline cli/verbs.go:96]: can we please specify the path via the config file?

Comment (walterchris) [reply cli/verbs.go:96]: Understood why this needs to be in a special location. Alright.

**Review by walterchris — DISMISSED** [review]

File: cli/verbs.go (hunk 1 above)
Comment (glabelderp) [reply cli/verbs.go:96]: We could do it via an environment variable, that is set inside the container, and if it is not set default to a hardcoded file.

**Comment by glabelderp** [comment]
My plan was that once this is merged other firmwareci related projects use the constants under /pkg/env/env.go to set the correct Environmental Variable.

**Review by llogen — APPROVED** [review]
LGTM

**Comment by walterchris** [comment]
Follow-up: use `pkg/env/env.go` in firmwareci

**Pull request merged**